common/app/common/commercial/hosted/hardcoded/RenaultHostedPages.scala

```diff
@@ -28,7 +29,7 @@ object RenaultHostedPages {
     val videoTitle = "Designing the car of the future"
     HostedVideoPage(
       campaign,
-      pageUrl = "https://www.theguardian.com/commercial/advertiser-content/renault-car-of-the-future/design-competition-teaser",
+      pageUrl = s"$host/commercial/advertiser-content/renault-car-of-the-future/design-competition-teaser",
```

Comment:
It should be https, but may be dangerous to change it sitewide at the moment.  So I think this is fine.